svg.js.d.ts

```diff
@@ -144,46 +614,421 @@ declare module "@svgdotjs/svg.js" {
         h: number;
         x2: number;
         y2: number;
-        merge(box: Box): Box;
-        transform(m: Matrix): Box
+
+        constructor();
         constructor(source: string);
-        constructor(source: []);
-        constructor(source: object);
+        constructor(source: number[]);
+        constructor(source: DOMRect);
         constructor(x: number, y: number, width: number, height: number);
+
+        merge(box: Box): Box;
+        transform(m: Matrix): Box
         addOffset(): this;
         toString(): string;
         toArray(): number[];
         isNulled(): boolean;
+        to(v: MorphValueLike): Morphable;

```

Comment:
Actually you can only morph a Box to a boxlike.
The other `to` functions are all with any but as a rule of thumb you can say: Everything what goes for the constructor also goes for the `to` method of that class

Reply:
OK, I was confused about this and just trying to get the test to compile. I'll fix it